x-pack/plugins/lens/public/xy_visualization/xy_config_panel/axis_settings_popover.tsx

```diff
@@ -241,7 +242,7 @@ export const AxisSettingsPopover: React.FunctionComponent<AxisSettingsPopoverPro
       !inclusiveZeroError &&
       !boundaryError &&
       localExtent &&
-      localExtent !== debouncedExtent
```

Comment:
This was false even when the two weren't different in any meaningful way. This led to a state update that messed with applying a suggestion. Let me know if this raises a flag and you'd like more context/demo.